tests/integration.rs

```diff
@@ -6728,6 +6748,46 @@ mod common_parallel {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    #[cfg(not(feature = "mshv"))]
+    fn test_tpm() {
+        let focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
+        let guest = Guest::new(Box::new(focal));
+
+        let (mut swtpm_command, swtpm_socket_path) = prepare_swtpm_daemon(&guest.tmp_dir);
+
+        let mut guest_cmd = GuestCommand::new(&guest);
+        guest_cmd
+            .args(["--cpus", "boot=1"])
+            .args(["--memory", "size=512M"])
+            .args(["--kernel", fw_path(FwType::RustHypervisorFirmware).as_str()])
+            .args(["--tpm", &format!("socket={swtpm_socket_path}")])
+            .capture_output()
+            .default_disks()
+            .default_net();
+
+        //start swtpm daemon
+        let mut swtpm_child = swtpm_command.spawn().unwrap();
+        thread::sleep(std::time::Duration::new(2, 0));
+        let mut child = guest_cmd.spawn().unwrap();
+        let r = std::panic::catch_unwind(|| {
+            guest.wait_vm_boot(None).unwrap();
+            assert_eq!(
+                guest.ssh_command("ls /dev/tpm0").unwrap().trim(),
+                "/dev/tpm0"
+            );
```

Comment:
If you are planning to test the tpm module in the guest like what's described in the [tpm.md](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/tpm.md#testing), you can refer this document to include the `tpm2-tss` package in our custom guest image: https://github.com/cloud-hypervisor/cloud-hypervisor/blob/main/docs/custom-image.md.

Reply:
@praveen-pk Since the storage bucket is access controlled I can create that updated image for you.

Reply:
@praveen-pk Rob's PR #5117 just landed and it updated jammy image with `tpm2-tools` installed. You can use this updated image in your integration test by simply switching to `UbuntuDiskConfig::new(JAMMY_IMAGE_NAME.to_string())`.

Reply:
Thanks @rbradford! I added 2 more tests while using the customized jammy image.